doc/api/repl.md

```diff
@@ -909,20 +909,22 @@ to connect to both Unix and TCP sockets.
 By starting a REPL from a Unix socket-based server instead of stdin, it is
 possible to connect to a long-running Node.js process without restarting it.
 
-For an example of running a "full-featured" (`terminal`) REPL over
-a `net.Server` and `net.Socket` instance, see:
-<https://gist.github.com/TooTallNate/2209310>.
-
-For an example of running a REPL instance over [`curl(1)`][], see:
-<https://gist.github.com/TooTallNate/2053342>.
-
-This example is intended purely for educational purposes to demonstrate how
-Node.js REPLs can be started using different I/O streams.
-It should **not** be used in production environments or any context where security
-is a concern without additional protective measures.
-If you need to implement REPLs in a real-world application, consider alternative
-approaches that mitigate these risks, such as using secure input mechanisms and
-avoiding open network interfaces.
+### Examples
+
+* running a "full-featured" (`terminal`) REPL over
+  a `net.Server` and `net.Socket` instance:
+  <https://gist.github.com/TooTallNate/2209310>.
```

Comment:
I wonder if it still makes sense to link to an external gist, maybe we should copy the code over (not sure how it's licensed though it seems short enough to not fall under copyrightable code).

Reply:
I totally agree, an external gist to a personal github user feels off to me :confused:

Reply:
IMO, We should create our own example derived from the sample and still link over to the gist for credit.

Reply:
I can definitely do that :slightly_smiling_face::+1:

Reply:
done :slightly_smiling_face:  https://github.com/nodejs/node/pull/57762/commits/0c1abbece2430d5ff7770128d837867cb412fa68

Reply:
Ah yeah, one last question! :slightly_smiling_face: 

Do you think that the `multi REPL`  section should also be moved under the new `examples` section? :slightly_smiling_face: